src/content/reference/manifest.mdx

```diff
@@ -610,6 +610,34 @@ volumes:
 
 Sets the working directory of your development container.
 
+### forward ([string], optional)
+
+Port forwards activated on any `okteto up` command, independently of the dev container name.
+
+As in [forward](#forward-string-optional) in dev section, several formats are accepted for this, but it is necessary to indicate the name of the service to be forwarded using the `localPort:remoteService:remotePort` notation.
+
+```yaml
+forward:
+  - 5432:postgres:5432
+```
+
+You can also indicate the service to expose using a label selector.
+
+```yaml
+forward:
+  - localPort: 8080
+    remotePort: 80
+    name: app
+  - localPort: 5432
+    remotePort: 5432
+    labels:
+      app: db
+```
+
+Using a global forward, Okteto handles port collision if two `okteto up` sessions are running at the same time. 
```

Comment:
I think should be the first thing in the section, since it explains how this is different from the regular forward

Reply:
and remove sentence `Port forwards activated on any okteto up command, independently of the dev container name.`? I think they both express the same thing.

Reply:
yeah let's do that.